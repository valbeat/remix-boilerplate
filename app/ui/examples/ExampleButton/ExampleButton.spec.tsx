import { composeStories } from "@storybook/react";
import { render } from "@testing-library/react";
import * as Stories from "./ExampleButton.stories";

const { Default } = composeStories(Stories);

it("renders correctly", () => {
	const { asFragment } = render(<Default />);
	expect(asFragment()).toMatchSnapshot();
});
