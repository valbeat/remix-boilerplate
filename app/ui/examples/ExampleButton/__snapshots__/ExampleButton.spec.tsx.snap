// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<DocumentFragment>
  <button
    class="bg-blue-500 text-white font-bold py-2 px-4 rounded"
    type="button"
  >
    Example Button
  </button>
</DocumentFragment>
`;
